mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.js

```diff
@@ -309,6 +324,64 @@ class ShowArtifactLoggedModelView extends Component {
     );
   }
 
+  renderMlflowSparkCodeSnippet() {
+    const { runUuid, path } = this.props;
+    const modelPath = `runs:/${runUuid}/${path}`;
+    return (
+      <>
+        <Title level={3}>
+          <FormattedMessage
+            defaultMessage='Make Predictions'
+            // eslint-disable-next-line max-len
+            description='Heading text for the prediction section on the registered model from the experiment run'
+          />
+        </Title>
+        <div className='artifact-logged-model-view-code-content'>
+          <div css={styles.item}>
+            <Paragraph
+              dangerouslySetAntdProps={{
+                copyable: { text: this.mlflowSparkCodeText(modelPath) },
+              }}
+            >
+              <pre style={{ wordBreak: 'break-all', whiteSpace: 'pre-wrap', marginTop: 10 }}>
+                <div className='code'>
+                  <span className='code-keyword'>import</span> mlflow{`\n`}
+                  logged_model = <span className='code-string'>{`'${modelPath}'`}</span>
```

Comment:
Looks like we're duplicating a lot of code here. Can we factor it out?

Reply:
The duplicate code is the three lines of code - make predictions, import, assign log model
<img width="660" alt="Screen Shot 2022-10-03 at 11 16 44 AM" src="https://user-images.githubusercontent.com/12734110/193649452-792de313-5bf6-41c7-9cc9-ace17686bef4.png">

And it's duplicating only a small part of 1/2 of renderPyfuncCodeSnippet. I feel it's cleaner to keep them separate, how you feel?

Reply:
Oh I'll have to add back "pyfunc.load_model". Then that's a lot of duplicate code. I'll try to simplify.

Reply:
Only moved the pandas frame part to a method:(